js/Control.OSMGeocoder.js

```diff
@@ -19,7 +19,7 @@ L.Control.OSMGeocoder = L.Control.extend({
 			this._map.fitBounds(bounds);
 
 			// IDE - Map have the power to draw a marker
-			RoadMap.addMarkerSearch(this._map.fitBounds(bounds));
+			RoadMap.setRequestedPositionMarker(this._map.fitBounds(bounds));
```

Comment:
You should not modify vendor libraries as they could be updated.
Actually, as soon as you modify a library, you have to care about its updates to merge your modification at any time, i.e. you have to maintain it. So we better avoid it!